### DISCUSSION TITLE
stream (stdout vs stderr) info is missing from kubernetes logs. Why?

### DISCUSSION THREAD

**Discussion by reegnz** [fluent/fluent-operator#1242] [Q&A]
I'm noticing that stream info is dropped with the kubernetes filter bundled with the helm chart. Couldn't find any reason for that design decision, so could someone explain why that is like that?

Also if there is no good reason to drop it, could it be added back in? It's quite valuable information.

See: https://github.com/fluent/fluent-operator/blob/e0d936ddfe45c8588159fd436e47f897457f6d26/charts/fluent-operator/templates/fluentbit-clusterfilter-kubernetes.yaml#L31

**Comment by benjaminhuo** [comment]
The removed entries are to reduce storage costs.
